test/integration/emitter_test.cpp

```diff
@@ -642,6 +642,23 @@ TEST_F(EmitterTest, Null) {
   ExpectEmit("- ~\n- null value: ~\n  ~: null key");
 }
 
+TEST_F(EmitterTest, NullStyles) {
+  out << BeginSeq
```

Comment:
This test would read better as a "table" test, where you have a small data table that you then iterate over to test each case in succession.

See, e.g., https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

You could try this explicitly in GoogleTest:

https://chromium.googlesource.com/external/github.com/google/googletest/+/HEAD/googletest/docs/advanced.md#value_parameterized-tests

but I've never used that so I don't know how easy it is. A for loop would be fine also.

Reply:
I've used Google Test's parameterised testing support before, but I wrote the tests to fit the style of the surrounding tests. Can you point me to an existing example in the `yaml-cpp` tests so I can match whatever the new style is supposed to be?

Reply:
Never mind, there are no existing examples and it's less clear what to do here than I had thought. This is fine.